### ISSUE TITLE
Drive temperature broken

### ISSUE THREAD

**Issue by Ubsefor** [007revad/Synology_HDD_db#366]
The script somehow breaks synology drive temperature reporting, it gets stuck at 29C constantly, while smartmontools report correct temperature

**Comment by 007revad** [comment]
Do you mean smartctl?

Which Synology model and DSM version?

**Comment by Ubsefor** [comment]
> Do you mean smartctl?

Yes, smartctl -a /dev/block | grep "Temp" shows correct temperature, its also shown correctly in "scrutiny"
Right now I’m patching synodrivedb.db by hand, we'll see if that will help

> Which Synology model and DSM version?

SA6400, 7.2.2-72806

**Comment by 007revad** [comment]
syno_hdd_db only edits model.dtb if the script detects a E10M20-T1, M2D20, M2D18 or M2D17 connected to a NAS model that does not officially support them and the E10M20-T1, M2D20, M2D18 or M2D17 is not already in model.dtb.

So the script would not touch model.dtb on a SA6400
